lib/help_scout/thread.rb

```diff
@@ -3,12 +3,30 @@
 module HelpScout
   class Thread < HelpScout::Base
     class << self
+      def create(conversation_id, thread_type, params)
+        HelpScout.api.post(
+          create_path(conversation_id, thread_type),
+          HelpScout::Util.camelize_keys(params)
+        )
+      end
```

Comment:
We're not in love with the approach, but [we've used `true` for the return value](https://github.com/taxjar/help_scout-sdk/blob/b951ca2f6d99baeb958b5f2dbb1ff0139f4917dc/lib/help_scout/attachment.rb#L11) for other objects that only return a 204 header upon creation. I think we should do the same here rather than exposing the response object. Non-successful responses should `raise` so we should be covered there.

I've considered using a configuration option to specify that you'd like created objects hydrated. I'm hesitant to add the extra request without user opt-in, but I can certainly see the appeal of receiving back the object rather than a boolean/the location etc.

Reply:
Sweet! Configuration option would be cool, but I don't think it's ultra critical or anything. I was mostly wanting the full object back while I was working on this, and meant to switch it to return true like you do in other places.